nutrinator/generator.py

```diff
@@ -48,3 +48,70 @@ def __generate_recipes(self, allowed: List[int]) -> np.ndarray:
     def __generate_portions(self, size: int) -> np.ndarray:
         return self.generator.choice(self.config.portion_set, (size, *self.shape), replace=True)
 
+    def generate_neighbour(self, recipes: np.ndarray, portions: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
+        neighbour = self.__generate_neighbour(recipes, portions)
```

Comment:
If I see correctly, this method generate neighbour in place, thus overriding old result